### PR TITLE
VMs: add doc test, re-org examples

### PR DESCRIPTION
**Pull request by howardjohn**
Currently, we have 3 examples:
* single network
* multinetwork, which is identical to single network
* bookinfo

These all do the same thing. None of them are docs-tested.

This PR refactors these into a single example. All we need to show is calls from k8s -> VM and VM -> k8s, so 3 samples are not really needed.

The single final example now has a docs tests written for it as well.

### CODE REVIEW

**Review by stevenctl — APPROVED** [review]
This looks great

**Comment by nmittler** [comment]
@howardjohn let me know when you're ready and I'll approve.

**Comment by howardjohn** [comment]
Its ready to go as soon as the `doc.test.profile_none_istio.io ` test passes - which I am hoping will happen this time

File: content/en/docs/examples/virtual-machines/index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ You can find details of configuring MySQL at [Mysql](https://mariadb.com/kb/en/l
 On the VM add ratings database to mysql.
```
Comment (ericvn) [inline content/en/docs/examples/virtual-machines/index.md:69]: Maybe (and I know this wasn't changed), `On the VM, add a ratings database to mysql`.

File: content/en/docs/examples/virtual-machines/index.md [hunk 2]
```diff
@@ -97,36 +97,34 @@ $ mysql -u root -ppassword test -e  "update ratings set rating=1 where reviewid=
...
-W1108 20:17:54.816122   40419 register.go:138] Got 'endpoints "mysqldb" not found' looking up endpoints for 'mysqldb' in namespace 'vm', attempting to create them
-I1108 20:17:54.886657   40419 register.go:180] No pre existing exact matching ports list found, created new subset {[{10.150.0.5  <nil> nil}] [] [{mysql 3306 }]}
-I1108 20:17:54.959744   40419 register.go:191] Successfully updated mysqldb, now with 1 endpoints
-{{< /text >}}
-
-Note that the 'mysqldb' virtual machine does not need and should not have special Kubernetes privileges.
-
 ## Using the mysql service
 
 The ratings service in Bookinfo will use the DB on the machine. To verify that it works, create version 2 of the ratings service that uses the mysql db on the VM. Then specify route rules that force the review service to use the ratings version 2.
```
Comment (ericvn) [inline content/en/docs/examples/virtual-machines/index.md:124]: use the DB on the **virtual** machine

or maybe change to `VM` as we use that in the next sentence.

**Review by ericvn — COMMENTED** [review]
One thing that is missing from the removed pages is showing the sending of a request to the VM from Kubernetes. Granted, we see it's done implicitly in the example as we are calling the mysqldb service, but we no longer show it explicitly.

**Comment by stevenctl** [comment]
fixes https://github.com/istio/istio.io/issues/5234

**Comment by howardjohn** [comment]
Test is passing now! failure is from an existing issue with an unrelated test

**Review by nmittler — APPROVED** [review]

**Pull request merged**